models/tracker/src/migration.ts

```diff
@@ -387,11 +387,183 @@ async function upgradeIssues (tx: TxOperations): Promise<void> {
   }
 }
 
+async function renameSprintToMilestone (client: MigrationClient): Promise<void> {
+  await client.update(
+    DOMAIN_TRACKER,
+    {
+      _class: tracker.class.Issue,
+      sprint: { $exists: true }
+    },
+    {
+      $rename: { sprint: 'milestone' }
+    }
+  )
+  await client.update(
+    DOMAIN_TRACKER,
+    {
+      _class: 'tracker:class:Sprint' as Ref<Class<Doc>>
+    },
+    {
+      _class: tracker.class.Milestone
+    }
+  )
+  const milestones = await client.find(DOMAIN_TRACKER, { _class: tracker.class.Milestone })
```

Comment:
Why ? Just remove objectId filter and it will rename all objectClass to objectClass.